### PR TITLE
2.0.0

### PR DESCRIPTION
**Pull request by legobeat**
This is the release candidate for version 2.0.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Added
+- Add `verify-snaps` script to verify snaps in the registry ([#67](https://github.com/MetaMask/snaps-registry/pull/67)) ([#80](https://github.com/MetaMask/snaps-registry/pull/80))
```
Comment (Mrtenz) [inline CHANGELOG.md:11]: This is not part of the published package, so I don't think you need to mention this.

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Changed
+- **BREAKING**: snaps `id` is now required to start with the prefix `npm:` ([#75](https://github.com/MetaMask/snaps-registry/pull/75))
```
Comment (Mrtenz) [inline CHANGELOG.md:11]:
```suggestion
- **BREAKING**: Snap IDs are now required to start with the prefix `npm:` ([#75](https://github.com/MetaMask/snaps-registry/pull/75))
```

**Review by Mrtenz — CHANGES_REQUESTED** [review]

**Review by Mrtenz — APPROVED** [review]

**Comment by FrederikBolding** [comment]
While this is technically a breaking change in the type I don't think it's really necessary to do a major bump for it. What do you think?

**Comment by Mrtenz** [comment]
> While this is technically a breaking change in the type I don't think it's really necessary to do a major bump for it. What do you think?

If it's breaking, technically or not, I think we should do a major bump. 😅

**Comment by FrederikBolding** [comment]
> This does in fact change runtime behavior so it's breaking even if we disregard that IMO

Yeah, you are right. However, for our use we do not currently rely on the runtime behaviour. Which is why I brought it up.

But I am fine with following semver properly here as well. Just wanted to voice the option, because it wouldn't be breaking with our usage.

**Pull request merged**